tests/CMakeLists.txt

```diff
@@ -28,9 +28,12 @@ endif()
 # generate config
 configure_file("${PROJECT_SOURCE_DIR}/tests/config.h.in" "${PROJECT_BINARY_DIR}/tests/config.h" ESCAPE_QUOTES @ONLY)
 
+SET(CLEAR_ALL_COMMAND rm -rf ${PROJECT_BINARY_DIR}/test_repositories /dev/shm/_tests_sr_*)
```

Comment:
Please use `set` instead of `SET` for consistency. Also, I think I would prefer variable name `TEST_CLEAN_COMMAND` to be more specific and go with the custom target.